oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandler.java

```diff
@@ -308,6 +314,15 @@ public APIGatewayProxyResponseEvent handleRequest(
                             accountInterventionService.getAccountStatus(
                                     userInfo.getSubject().getValue());
                     reproveIdentity = accountStatus.reproveIdentity();
+
+                    if (accountStatus.blocked() || accountStatus.suspended()) {
+                        return logoutService.handleAccountInterventionLogout(
```

Comment:
This doesn't quite look right to me. If they're on an ID journey we proceed to IPV for a suspend